pyccel/codegen/printing/cucode.py

```diff
@@ -131,4 +136,71 @@ def _print_ModuleHeader(self, expr):
                           global_variables,
                           function_declaration,
                           "#endif // {name.upper()}_H\n"))
+    def _print_Allocate(self, expr):
+        variable = expr.variable
+        if not isinstance(variable.class_type, CudaArrayType):
+            return super()._print_Allocate(expr)
```

Comment:
/bot accept
There's no need to test that because I call print_allocate in case the allocation is something related to C, just to make sure all the stuff is working well in CUDA